conftest.py

```diff
@@ -0,0 +1,8 @@
+"""Project conftest"""
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def disable_webpack(settings):
```

Comment:
Can you rename this to something more general like `test_suite_default_settings` or something like that?

Reply:
Sure, I'll do `default_settings` so it doesn't start with `test`